File: app/src/main/java/com/weiwa/ljl/weiwa/OnUserUpdatedListener.java
package com.weiwa.ljl.weiwa;

import com.weiwa.ljl.weiwa.WeiboModel.WeiboPojo;

/**
 * Created by hzfd on 2017/2/13.
 */
public interface OnUserUpdatedListener {
    void onUpdate(WeiboPojo pojo, int updateType);
}

File: app/src/main/java/com/weiwa/ljl/weiwa/listener/OnUserUpdatedListener.kt
package com.weiwa.ljl.weiwa.listener

import com.weiwa.ljl.weiwa.network.WeiboPojo

/**
 * Created by hzfd on 2017/2/13.
 */
interface OnUserUpdatedListener {
    fun onUpdate(pojo: WeiboPojo, updateType: Int)
}
